compiler/rustc_middle/src/query/mod.rs

```diff
@@ -2293,6 +2293,19 @@ rustc_queries! {
         desc { "whether the item should be made inlinable across crates" }
         separate_provide_extern
     }
+
+    /// Returns the list of missing features for a call to a callee of a certain type in a certain
+    /// caller (identified by DefId).
+    query call_site_abi_missing_features(key: (Ty<'tcx>, DefId)) -> &'tcx Vec<String> {
+        arena_cache
+        desc { "missing features for ABI compatibility at call sites" }
+        cache_on_disk_if { key.1.is_local() }
```

Comment:
Oh I only just saw this... yeah this makes little sense. We often monomorphize functions from other crates, after all.